data-prepper-core/src/main/java/com/amazon/dataprepper/ContextManager.java

```diff
@@ -15,15 +15,13 @@
  * <p>Creates Spring {@link org.springframework.context.ApplicationContext} hierarchy for Dependency Injection with limited visibility.</p>
  * <p>
  *     Application Context Hierarchy
- *     <pre>
  *         Public Application Context<br>
  *         ├─ Core Application Context<br>
  *         ├─ Shared Plugin Application Context<br>
  *             ├─ Plugin Isolated Application Context<br>
- *     </pre>
  * </p>
  */
-class ContextManager {
+public class ContextManager {
```

Comment:
Any context for why this needs to be public?

Reply:
I need access to it for FizzyDrPepper in order to spin up a Data Prepper instance